### PR TITLE
perf: File path jump function

### PR DESCRIPTION
**Pull request by fit2bot**
perf: File path jump function

### CODE REVIEW

File: ui/src/components/pamFileList/components/fileManage/index.vue [hunk 1]
```diff
@@ -150,6 +282,17 @@ const rowProps = (row: RowData) => {
...
+
+      if (suffix !== 'Folder') {
+        return message.error('暂不支持文件预览');
+      }
+
+      mittBus.emit('change-path', { path: splicePath });
+      disabledBack.value = false;
     }
   };
 };
```
Comment (fit2cloudrd) [inline ui/src/components/pamFileList/components/fileManage/index.vue:298]: This code has been reviewed for correctness but there can be some optimizations and clarifications needed. Some points to consider:

1. In the button components used in `NButton` and `n-flex`, make sure they have proper font sizes (e.g., not too small) which could result in problems such as unreadable elements if smaller.
   
2. The table component's column data might need adjustment since the columns being passed should match exactly when it renders.

3. Use consistent styles across all n-components, especially inline ones like `<ArrowForwardIosFilled>`.

4. Check other parts of the codebase where similar design decisions are made so that consistency can be maintained. This may include CSS rules, colors, etc.

5. Since this involves React and Vue components, please note that optimizing these would involve understanding how their internal state management works and how you modify that internally. For instance, reusing props instead of repeating values or simplifying conditional logic could yield improvements over repetition.

6. Please remember that without actual implementation details of any libraries involved, we cannot give specific advice beyond general recommendations here.

In summary, this review suggests looking at areas where uniformity exists and apply those principles consistently. If there isn't good reason for certain patterns, then either adapt them properly or remove redundancy.

File: ui/src/components/pamFileList/index.vue [hunk 2]
```diff
@@ -388,39 +395,39 @@ const createColumns = (): DataTableColumns<RowData> => {
...
+    //               component: unref(item.iconName)
+    //             })
+    //           ]
+    //         }
+    //       );
+    //     });
+    //   }
+    // }
   ];
 };
```
Comment (fit2cloudrd) [inline ui/src/components/pamFileList/index.vue:433]: No major inconsistencies found. However, a few minor points:

1. The `folder` icon should be aligned with other elements rather than being positioned inside of them.
2. It would help to maintain consistency across different parts of codebase when it comes to the styles used to align items.

Regarding optimizations, you can use the following approaches:
- Consider using ES modules and dynamic imports instead of global namespace ones to reduce memory usage.
- Extract common functions into separate utility components that are reused throughout your application (e.g, createColumn, handleCheckboxClick).
- If necessary, use TypeScript/JSX to improve static typing, which might prevent some bugs down the line.

Remember, my feedback is based entirely on what I consider "standard" best practices given current day specifications.

Please review these changes yourself as needed after applying them.

File: ui/src/hooks/useFileManage.ts [hunk 3]
```diff
@@ -159,6 +177,10 @@ export const useFileManage = () => {
       mittBus.on('file-refresh', () => {
         refresh(<WebSocket>socket);
       });
+
+      mittBus.on('change-path', (path: string) => {
+        handleChangePath(<WebSocket>socket, path);
+      });
     }
   }
```
Comment (fit2cloudrd) [inline ui/src/hooks/useFileManage.ts:186]: There is no difference between original file management code from the previous year's dataset and the updated one from 2024 because it seems that there may have been some changes made to this code but with insignificant modifications or adjustments, as the code appears to work correctly without any errors or noticeable deviations.
